Controllers/UploadController.py

```diff
@@ -10,14 +10,14 @@
 def uploadNewFile():
     try:
         if ("SID" not in request.cookies):
-            return Response("No session detected", status=400)
+            return Response("No session detected", status=401)
         if ('file' not in request.files):
-            return Response("No files were uploaded", status=400)
+            return Response("No files were uploaded", status=402)
```

Comment:
This should probably stay as a 400

Reply:
Oops I forgot to change them back, were using it for debugging